docs/developing/publishing-apps.md

```diff
@@ -0,0 +1,105 @@
+
+### Metadata guide
+
+#### icons
+
+You can embed both SVGs or PNGs, and Sandstorm will use the best version provided for the use in question.
+
+* The `appGrid` icon represents your app on the "New" screen on Sandstorm. It should be 128 x 128 pixels, and no larger than 64 KB.
+* The `grain` icon represents individual grains on both the navbar and the grain list. It should be 24 x 24 pixels, and no larger than 4 KB. If you omit this, the appGrid icon will be used.
+* The `market` icon is used in the app market. It should be 150 x 150 pixels, and no larger than 256 KB. If you omit this, the appGrid icon will be used.
+* The `marketBig` icon is used on an app's specific page on the app market. It should be 300 x 300 pixels, and no larger than 256 KB. If you omit this, the market icon will be used (raster images may look bad).
+
+#### website
+
+This should be the app's main website URL.
+
+#### codeUrl
+
+This should be the URL of the app's source code repository, like Github. It is recommended if there is a repository specific to the Sandstorm package, you utilize that one, rather than the upstream one. This field is mandatory if you utilize a license that requires redistributing code, like the GPL, but is optional otherwise.
+
+#### license
+
+This is how you will specify the license under which you are distributing the app. The default is `none`, which conveys no rights of redistribution to the user.
+
+Currently, the following open source licenses are recognized: `mit`, `apache2`, `gpl3`, `agpl3`, `bsd3Clause`, `bsd2Clause`, `gpl2`, `lgpl2`, `lgpl3`, `isc`, `artistic2`, `python2`, `php3`, `mpl2`, `cddl`, `epl`, and `cpal`. If you need to add an additional license, you can open an issue or submit a pull request.
+
+You can select `openSource` and specify an OSI-approved license, or select `proprietary` and embed the full text of the license. If you choose a proprietary license, which may contain more restrictive permissions, Sandstorm will display the license to the user and have them accept it before they are able to use the app. If your app does not contain more restrictive permissions, you may consider releasing under `none` to avoid this. Finally, `publicDomain` is also available, but it is preferable to use a permissive open source license instead.
+
+You may also need to include `notices` if your app is required to display any third-party copyright notices, for example due to use of third-party open source libraries.
+
+#### categories
+
+You may select the market categories to which your app belongs. You may select multiple, but you may be asked to make changes if the market moderators feel they are inappropriate.
+
+Currently, the following categories are accepted: productivity, communications, social, webPublishing, office, developerTools, science, graphics, media, games, and other.
```

Comment:
I'd prefer a bulleted list here, and/or to have the words be surrounded by `backticks` to make it clear they are `literal values`, but I can deal with whatever.